docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-source.md

```diff
@@ -0,0 +1,144 @@
+
+The Druva Source requires you to provide a **Client ID**, **Client Secret Key**, **Base URL**. To get these, follow the instructions from [Druva documentation](https://developer.druva.com/reference/reference-getting-started).
+
+
+## Data Sources
+
+The Druva API integration consumes event logs from the [Data Governance Cloud](https://developer.druva.com/reference/get_eventmanagement-v2-events) or [Data Governance GovCloud](https://developer.druva.com/reference/get_eventmanagement-v2-events-1) and sends it to Sumo Logic.
+
+
+## States
+
+A Druva API integration Source is an integrated Security Awareness Training and Simulated Phishing platform that helps to train users to understand the dangers of spam, phishing, spear phishing, malware, ransomware, and social engineering through simulated phishing and security awareness training.
+
+When a Druva API Source is created, it goes through the following states:
+
+1. **Pending**. Once the Source is submitted, it is validated, stored, and placed in a **Pending** state.
+1. **Started**. A collection task is created on the Hosted Collector.
+1. **Initialized**. The task configuration is completed in Sumo Logic.
+1. **Authenticated**. The Source is successfully authenticated with Druva.
+1. **Collecting**. The Source is actively collecting data from Druva.
+
+If the Source has any issues during any one of these states, it is placed in an **Error** state.
+
+When you delete the Source, it is placed in a **Stopping** state. When it has successfully stopped, it is deleted from your Hosted Collector.
+
+On the Collection page, the [Health](/docs/manage/health-events#collection-page) and Status for Sources is displayed. Use [Health Events](/docs/manage/health-events) to investigate issues with collection. <br/>![C2C error icon on collection page.png](/img/send-data/C2C-error-icon-on-collection-page.png)
+
+Hover your mouse over the status icon to view a tooltip with a count of the detected errors and warnings.<br/> ![hover c2c error.png](/img/send-data/hover-c2c-error.png)
+
+## Setup and configuration Source
+
+When you create a Druva Source, you add it to a Hosted Collector. Before creating the Source, identify the Hosted Collector you want to use or create a new Hosted Collector. For instructions, see [Configure a Hosted Collector](/docs/send-data/hosted-collectors/configure-hosted-collector.md).
+
+**To configure a Druva Source**
+
+1. In Sumo Logic, select **Manage Data** > **Collection** > **Collection**.
+1. On the **Collectors** page, click **Add Source** next to a Hosted Collector.
+1. Select **Druva**. <br/><img src={useBaseUrl('img/send-data/druva-icon.png')} alt="druva-icon.png" width="200"/>
```

Comment:
Broken image. It looks like image is in the docs folder  (`/docs/send-data`). Please move it to `/static/img/send-data`.